frontend/src/components/eam/eam.tsx

```diff
@@ -58,6 +69,37 @@ const EAM: FC = () => {
     );
   };
 
+  const handleBackPage = (): void => {
+    const forwardPage = currentPage - 1;
+    if (forwardPage !== 0) {
+      setCurrentPage(forwardPage);
+      const forwardFrom = from - 5;
+      setFrom(forwardFrom);
+      dispatch(
+        eamActions.loadWorkers({
+          tenantId: tenantId as string,
+          from: forwardFrom,
+          count: 5,
+        }),
+      );
```

Comment:
This code used twice in this file, could we just move it into different function like `handleLoadWorkers(tenantId, from, count)`